datetime/export_test.go

```diff
@@ -0,0 +1,5 @@
+package datetime
```

Comment:
Shouldn't it be some `datetime_test` or it won't work?

Reply:
It's ok for export_test.go, see as examples:

https://go.dev/src/net/http/export_test.go
https://github.com/tarantool/go-tarantool/blob/master/export_test.go